services/app-web/src/testHelpers/apolloMocks/healthPlanFormDataMock.ts

```diff
@@ -307,6 +307,118 @@ function mockStateSubmission(): LockedHealthPlanFormDataType {
     }
 }
 
+function mockContractAndRatesDraftWithDocuments(
```

Comment:
I don't think we need this since we can pass in partial package data into `mockContractAndRatesDraft`.